src/cmake/thirdparty/SetupAxomThirdParty.cmake

```diff
@@ -237,13 +237,30 @@ endif()
 #------------------------------------------------------------------------------
 if (LUA_DIR)
     include(cmake/thirdparty/FindLUA.cmake)
-    blt_import_library(
-        NAME          lua
-        INCLUDES      ${LUA_INCLUDE_DIR}
-        LIBRARIES     ${LUA_LIBRARY}
-        TREAT_INCLUDES_AS_SYSTEM ON
-        EXPORTABLE    ON)
+    if(NOT TARGET lua)
+        blt_import_library( NAME        lua
+                            LIBRARIES   ${LUA_LIBRARIES}
+                            INCLUDES    ${LUA_INCLUDE_DIR}
+                            EXPORTABLE  ON)
+    endif()
+
+    blt_print_target_properties(TARGET lua)
+    blt_print_variables(NAME_REGEX lua IGNORE_CASE)
+
```

Comment:
```suggestion
    blt_print_target_properties(TARGET lua)
    blt_print_variables(NAME_REGEX lua IGNORE_CASE)

```

Reply:
Thanks -- I meant to remove these or comment them out.